brian2/devices/cpp_standalone/templates/network.cpp

```diff
@@ -5,6 +5,9 @@
 #include<iostream>
 #include <ctime>
 #include<utility>
+#include<string>
+#include <sstream>
+
```

Comment:
I'm not 100% sure, but I think it is enough to have these includes in `main.cpp`
```suggestion
```

Reply:
yes, I missed removing # from network.cpp file